lib/std/process.zig

```diff
@@ -1831,7 +1831,7 @@ pub const CreateEnvironOptions = struct {
     /// `null` means to leave the `ZIG_PROGRESS` environment variable unmodified.
     /// If non-null, negative means to remove the environment variable, and >= 0
     /// means to provide it with the given integer.
-    zig_progress_fd: ?i32 = null,
+    zig_progress_fd: if (native_os == .windows) ?usize else ?i32 = null,
```

Comment:
Using usize for windows excludes possibility of negative number here.. not sure about the consequences of that yet.

Reply:
Isn't it very sketchy to use just a negative number to mean something special for one particular function? Should probably be handled with an enum or something?